### PR TITLE
chore: remove wallet alert on outcome, remove showOutcome function

### PR DESCRIPTION
**Pull request by katelynsills**
removes the "showOutcome" function

### CODE REVIEW

File: packages/dapp-svelte-wallet/ui/src/Transaction.svelte [hunk 1]
```diff
@@ -25,7 +25,6 @@
     if (typeof outcome !== 'string') {
       outcome = 'Offer was accepted.';
     }
-    alert(outcome);
```
Comment (dtribble) [inline packages/dapp-svelte-wallet/ui/src/Transaction.svelte:28]: Unfortunately I think we need something more complicated to keep encouragement working. Plausible is to only pop up if there was a string return, and then make the faucet not provide a string result value. But hacky. I'll just cope now.

We also should really remember that outcome with the offer in the wallet, so

Comment (katelynsills) [reply packages/dapp-svelte-wallet/ui/src/Transaction.svelte:28]: Currently the outcome is being sent back to the encouragement dapp. There's no reason to alert it in the wallet.

Comment (katelynsills) [reply packages/dapp-svelte-wallet/ui/src/Transaction.svelte:28]: See https://github.com/Agoric/dapp-encouragement/pull/73 for use of this functionality

File: packages/dapp-svelte-wallet/ui/src/Transaction.svelte [hunk 2]
```diff
@@ -25,7 +25,6 @@
     if (typeof outcome !== 'string') {
```
Comment (michaelfig) [inline packages/dapp-svelte-wallet/ui/src/Transaction.svelte]: This whole function should go away.  If you just want to silence a dapp in the meantime, use an offer where `offer.dappContext = true`.

File: packages/dapp-svelte-wallet/ui/src/Transaction.svelte (hunk 1 above)
Comment (michaelfig) [reply packages/dapp-svelte-wallet/ui/src/Transaction.svelte:28]: > See [Agoric/dapp-encouragement#73](https://github.com/Agoric/dapp-encouragement/pull/73) for use of this functionality

... which I've patched not to rely on this PR being merged.

**Review by michaelfig — APPROVED** [review]
LGTM!

**Pull request merged**